module/Photo/src/Photo/Service/Admin.php

```diff
@@ -178,7 +178,8 @@ public function upload($files, $album)
             }
         } else {
             throw new \Exception(
-                $translator->translate('The uploaded file is not a valid image')
+                $translator->translate("The uploaded file is not a valid image \nError: " .
+                    json_encode($imageValidator->getMessages()))
```

Comment:
I don't think `json_encode` returns a very readable error message. Perhaps something like `implode(',', array_values($imageValidator->getMessages()))` would be better?